lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/persistence/DBOMessageToUser.java

```diff
@@ -214,23 +226,71 @@ public MigrationType getMigratableTableType() {
 	// once this has run for a single release, the back up objects will no longer have a 'subject'
 	// field.  Then the translator can then be reverted to the simple, default version
 	@Override
-	public MigratableTableTranslation<DBOMessageToUser, DBOMessageToUser> getTranslator() {
-		return new MigratableTableTranslation<DBOMessageToUser, DBOMessageToUser>() {
+	public MigratableTableTranslation<DBOMessageToUser, DBOMessageToUserBackup> getTranslator() {
+		return new MigratableTableTranslation<DBOMessageToUser, DBOMessageToUserBackup>() {
 			@Override
-			public DBOMessageToUser createDatabaseObjectFromBackup(DBOMessageToUser backup) {
-				return backup;
+			public DBOMessageToUser createDatabaseObjectFromBackup(DBOMessageToUserBackup backup) {
+				DBOMessageToUser dbo = new DBOMessageToUser();
+				dbo.setMessageId(backup.getMessageId());
+				dbo.setRootMessageId(backup.getRootMessageId());
+				dbo.setInReplyTo(backup.getInReplyTo());
+				dbo.setSubjectBytes(backup.getSubjectBytes());
+				dbo.setSent(backup.getSent());
+				dbo.setNotificationsEndpoint(backup.getNotificationsEndpoint());
+				dbo.setProfileSettingEndpoint(backup.getProfileSettingEndpoint());
+				dbo.setWithUnsubscribeLink(backup.getWithUnsubscribeLink());
+				dbo.setWithProfileSettingLink(backup.getWithProfileSettingLink());
+				dbo.setIsNotificationMessage(backup.getIsNotificationMessage());
+				try {
+					if (backup.getTo() != null) {
+						dbo.setTo(backup.getTo().getBytes("UTF-8"));
+					}
+					if (backup.getCc() != null) {
+						dbo.setCc(backup.getCc().getBytes("UTF-8"));
+					}
+					if (backup.getBcc() != null) {
+						dbo.setBcc(backup.getBcc().getBytes("UTF-8"));
+					}
+				} catch (UnsupportedEncodingException e) {
+					throw new RuntimeException(e);
+				}
+				return dbo;
 			}
 			
 			@Override
-			public DBOMessageToUser createBackupFromDatabaseObject(DBOMessageToUser dbo) {
-				return dbo;
+			public DBOMessageToUserBackup createBackupFromDatabaseObject(DBOMessageToUser dbo) {
```

Comment:
This doesn't help us get to the state in which the backup is DBOMessageToUser.  Is your plan that DBOMessageToUserBackup would become a permanent part of the migration scheme?